### ISSUE TITLE
windows and the python3 binary

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#1010]
### bundle

not relevant

### python.org installer

3.8.2 does not install the `python3` binary

### Anaconda

todo: not tested yet

### misc

Win10 seems to ship a `python3` binary: running it opens the Microsoft Store at python installer.  haven't tried it yet

```

C:\Users\anon>where python
C:\Users\anon\AppData\Local\Programs\Python\Python38\python.exe
C:\Users\anon\AppData\Local\Microsoft\WindowsApps\python.exe

C:\Users\anon>where python3
C:\Users\anon\AppData\Local\Microsoft\WindowsApps\python3.exe

```

**Comment by cbm755** [comment]
"Microsoft Store" python provides both `python` and `python3`.

**Comment by mtmiller** [comment]
Yeah, I've noticed both the official installer and anaconda only ship python.exe, I don't know why they don't include python3.exe.

Isn't it likely that Windows users will have to set a full path to Python anyway, like

    setenv PYTHON 'C:\Program Files\Python3.8\python'

**Comment by cbm755** [comment]
Not if they click "Add python.exe to path" during the install of the installer python.org...

**Comment by mtmiller** [comment]
I have the official Python 3.8.1 and Anaconda 3 2019.10 (Python 3.7.4) installed on Windows 10, and both provide python.exe and pythonw.exe, there is no python3.exe.

**Comment by mtmiller** [comment]
If the official Octave builds are now going to start providing some version of Python and SymPy (per JohnD on the maintainers list), maybe that can be made to provide python3.exe, and it will always be in Octave's executable path.

**Comment by cbm755** [comment]
Oh I hope not!  Bundle is really just a kludge.

At any rate, it seems like "python3" is not commonly the correct binary for windows (except maybe when one uses the Microsoft Store), so I guess our current default isn't quite right...  Maybe #1009 and #1008 and so on are the way to go.
